lntest/timeouts.go

```diff
@@ -15,7 +15,7 @@ const (
 
 	// ChannelCloseTimeout is the max time we will wait before a channel is
 	// considered closed.
-	ChannelCloseTimeout = time.Second * 30
+	ChannelCloseTimeout = time.Second * 120
```

Comment:
we have own timeouts defined for darwin. Should we maybe also have custom timeouts for etcd runs?

Reply:
Any comment on this?

Reply:
Sorry missed your comment. Added separate timeouts for etcd.

Reply:
Looks good!